models/mento/celo/mento_celo_pools.sql

```diff
@@ -0,0 +1,37 @@
+{{
+  config(
+    tags = ['dunesql'],
+    schema = 'mento_celo',
+    alias = alias('pools'),
```

Comment:
We can also drop `alias()` functions in all config blocks.
```suggestion
    alias = 'pools',
```